### PR TITLE
[CPDNPQ-3181] NPQEYL contact data

### PR DESCRIPTION
**Pull request by alkesh**
### Context

Ticket: https://dfedigital.atlassian.net/browse/CPDNPQ-3181

### Changes proposed in this pull request

rake task to generate requested CSV

### CODE REVIEW

**Review by jebw — APPROVED** [review]
LGTM - couple of minor style suggestions :+1: 

One query I do have is if multiple specs were added at a later point - presumably they will not be compatible with `parallel_specs` since two specs running on different processes may end up concurrently writing to the same file?

Should that maybe be using a temp file some how in the `Rails.env.test?` scenario?

Definitely an edge case for a one off rake task so probably not significant enough to be worth worrying about

File: lib/tasks/one_off/eyl_users_3181.rake [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  def applications
+    Application.includes(:user, :school, :private_childcare_provider, :cohort, :declarations)
+      .where(course_id: Course.npqeyl.id, training_status: ["active", nil])
+  end
+
+  def started?(application)
+    !!application.declarations.find { |declaration| declaration.declaration_type == "started" }
+  end
+end
+EylUsers3181.new
```
[the remark this answers is not in the document]
Comment (alkesh) [reply lib/tasks/one_off/eyl_users_3181.rake:61]: you mean you want a whitespace line added?
for rake tasks that use a class, it's currently half-and-half if we have a whitespace line after the class definition.
